core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -83,36 +89,67 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
 
       <div ng-if="collection" class="oppia-exploration-list-table">
 
-        <table class="oppia-dashboard-table">
-          <tr>
-            <th>
-              <p>Exploration</p>
-            </th>
-            <th class="hidden-md hidden-sm hidden-xs">
-              <p>Objective</p>
-            </th>
-          </tr>
-          <tr ng-repeat="node in collection.getCollectionNodes()"
-              class="exploration-list-item">
-            <td class="oppia-exploration-list-table-item">
-              <a ng-href="/explore/<[node.getExplorationId()]>"
-                 ng-mouseover="updateExplorationPreview(node.getExplorationId())"
-                 ng-mouseleave="togglePreviewCard()"
-                 class="oppia-dashboard-list-summary">
-                <[node.getExplorationSummaryObject().title]>
-              </a>
-            </td>
-            <td class="hidden-md hidden-sm hidden-xs oppia-exploration-list-table-item">
-              <a ng-href="/explore/<[node.getExplorationId()]>"
-                 ng-mouseover="updateExplorationPreview(node.getExplorationId())"
-                 ng-mouseleave="togglePreviewCard()"
-                 class="oppia-dashboard-list-summary">
-                <[node.getExplorationSummaryObject().objective]>
-              </a>
-            </td>
-          </tr>
-        </table>
-
+        <svg width="500px" height="<[svgHeight + 'px']>" xmlns="http://www.w3.org/2000/svg">
+          <path fill="none"
+                stroke="#10301D"
+                stroke-dasharray="15,28"
+                stroke-linecap="round"
+                stroke-width="16"
+                d="M80 80 L220 80"/>
+          <path fill="none"
```

Comment:
Just noting that this still needs resolution -- and again, it might be worth generating them a single time when needed and updating them only when necessary (i.e. onResize).